### PR TITLE
SEO-171911 - AngularJS Control Missing h1

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Please review the changes for missing H1 below.

Regards,
Sharly A.

### CODE REVIEW

File: angularjs/Dialog/appearance-and-styling.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
+title: Appearance and styling in AngularJS Diagram Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline angularjs/Dialog/appearance-and-styling.md:3]: Control name is wrong, check and change.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: angularjs/Dialog/appearance-and-styling.md (hunk 1 above)
Comment (Sharly-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: Comment resolved

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

File: angularjs/Dialog/appearance-and-styling.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
+title: Appearance and styling in AngularJS Dialog Control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Sharly-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: modified

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**